src/nvim/if_cscope.c

```diff
@@ -1174,22 +1172,25 @@ static char *GetWin32Error(void)
 static int cs_insert_filelist(char *fname, char *ppath, char *flags,
                               FileInfo *file_info)
 {
-  short i, j;
+  size_t i;
+  bool empty_found;
 
-  i = -1;   /* can be set to the index of an empty item in csinfo */
```

Comment:
QB build complains that `i` is used uninitialized in `cs_insert_filelist`. I'm guessing it is because of this deletion.